packages/web-twig/tests/components-fixtures/textAreaDefault.twig

```diff
@@ -0,0 +1,13 @@
+<TextArea
+    id="example"
+    label="TextArea"
+    messsage="validation failed"
+    minlength="6"
+    maxlength="10"
+    name="example"
+    type="text"
```

Comment:
As i mentioned in web-react variant. Is there any reason, why is `type=` for `<textarea>`, or am I missing something here?

Reply:
@tomassychra You have a point. I think that this attribute should not be here. @dlouhak Please look at this closely but I think only the deletion of this line would be enough.

Reply:
Good eye! Yeah, the deletion of this line was enough. Thanks. 💪🏻